### PR TITLE
Token types

### PR DESCRIPTION
**Pull request by ddocs**

### CODE REVIEW

File: docs/learn/concepts/wallets.md [hunk 1]
```diff
@@ -3,35 +3,35 @@ title: Wallets
...
-Currently, (as of July 2022) it is not possible to use **Keplr Mobile App** with the Cudos Network. This is due to the limited functionality of the **Keplr App**. Stay tuned in [Cudos Discord](https://discord.com/invite/t397SKqf4u) for any updates. 
 :::
+In this way, multiple accounts can be set up on a single Wallet.
+:::
+
+## Wallet Accounts
+
+A **Wallet Account** consists of an **Address**, **Public Key** and **Private Key**.
+
+Each Wallet address corresponds to a single key pair that can always be reproduced deterministically from a mnemonic by knowing its derivation path. There are a number of standards used to generate key pairs from the same mnemonic e.g. BIP 49 or BIP 44.
```
Comment (nbaum) [inline docs/learn/concepts/wallets.md:30]: There's actually only one standard for computing things from mnemonic phrases: BIP39.

Using BIP39, a mnemonic phrase is turned into a seed, or master key. The other BIPs then use that seed to compute the actual private keys. BIP49 is not used on Cosmos. BIP44 is the one that Cosmos uses. [The scheme is described in more detail here](https://github.com/confio/cosmos-hd-key-derivation-spec)

**Review by nbaum — COMMENTED** [review]
You didn't ask me for feedback, but here it is.

File: docs/learn/concepts/accounts.md [hunk 2]
```diff
@@ -3,7 +3,86 @@ title: Accounts
...
+There are two keys: A **Public Key** and a **Private Key**. 
+
+The **Public Key** is used to generate an **Address** (or multiple addresses).
+
+An **Account object** is an implementation of an account keypair. It stores the following:
+
+1. Public Key (`PubKey`)
+2. `Address` derived from the Public Key
+3. Account number (`AccountNumber`)
+4. `Sequence`
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/accounts.md:21]: This is very interesting. Just reading up on replay attacks and sequence value or nonce. 

> The sequence number in the transaction has to match the sequence number on the chain.
Is this done in the code? 
What about timestamps? Perhaps I'm on a tangent here but Juno uses a tool for time synchronisation in relation to running a node: 
```
# install toolchain and ensure accurate time synchronization
sudo apt-get install make build-essential gcc git jq chrony -y
```

[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/accounts.md:21]: Separated account on chain from account on wallet.

File: docs/learn/concepts/accounts.md [hunk 3]
```diff
@@ -3,7 +3,86 @@ title: Accounts
...
+}
+```
+
+## Public key
+
+The Public Key - `PubKey` is safe to disclose and is used to verify a user or entity's identity.
+
+### Generation of Addresses
+
+`PubKey` is used to generate `Address` using algorithms. One or more unique addresses can be generated from the Public Key.
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/accounts.md:40]: This is super helpful for Validators to know or potential validators.
I'm realised there is a terminology issue with using the term Validator. This is used to describe both the entity or user who performs the validation process or validates on the network as well as a Validator node. This becomes extra confusing when describing Validator nodes as part of a Validator Cluster. 
Any thoughts appreciated.

File: docs/learn/concepts/accounts.md [hunk 4]
```diff
@@ -3,7 +3,86 @@ title: Accounts
...
+Signatures prove ownership and authorise changes, for example to funds used in a transaction. 
+The signature can be verified by another user or entity by confirming that its public key matches the public key of the sender. 
+
+This forms the basis of **Authentication** and **Assurance** on the network. 
+
+:::tip Security
+
+### Authentication 
+
+Authentication of a message or user/entity is guaranteed by the use of asymmetric public key cryptography. This is a cryptographic system that uses key pairs. The asymmetry refers to the fact that parties are not required to use a secure channel to exchange keys for encryption and decryption.
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/accounts.md:67]: Done

File: docs/learn/concepts/cryptography.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: Cryptography
+id: cryptography
+---
+
+Expand on accounts section of keys and cryptography used at Cudos.
+
+e.g. ECDSA, BIP 39 etc.
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/cryptography.md:8]: Wonderful yes, I'll add that to a separate branch. Trying to get these PRs smaller. :)

File: docs/learn/concepts/learn-tokens.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+title: Token types
+id: token-types
+---
+
+This section explains fungible and non-fungible token types for inter-blockchain and Ethereum. 
+
+## CW Tokens
+
+**cw20** is a specification for fungible tokens based on CosmWasm. The name and design is loosely based on Ethereum's ERC20 standard. cw20 tokens can be imported by smart contracts that wish to implement this specification, or by contracts that call to any standard cw20 contract.
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/learn-tokens.md:10]: done

File: docs/learn/concepts/wallets.md [hunk 7]
```diff
@@ -3,35 +3,35 @@ title: Wallets
...
-* Secret Network
-* Akash
-* Starname
-* Sifchain
-* CertiK
-* IRISnet
-* Regen Network
-* Cyber (beta)
-* Straightedge (beta)
+From this initial mnemonic, multiple key pairs can be derived with the Public key from each key pair used to derive an **address** (or multiple addresses).
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/wallets.md:20]: Done

File: docs/learn/concepts/wallets.md (hunk 1 above)
Comment (ddocs) [reply docs/learn/concepts/wallets.md:30]: done

File: docs/learn/concepts/accounts.md [hunk 8]
```diff
@@ -3,7 +3,86 @@ title: Accounts
...
+
+An account is stored as an **account object**.
+
+## Account Key Pairs
+
+There are two keys: A **Public Key** and a **Private Key**. 
+
+The **Public Key** is used to generate an **Address** (or multiple addresses).
+
+An **Account object** is an implementation of an account keypair. It stores the following:
```
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/accounts.md:16]: Done

File: docs/learn/concepts/accounts.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ddocs) [reply docs/learn/concepts/accounts.md:21]: Done

**Pull request merged**